browser-test/login.test.js

```diff
@@ -0,0 +1,76 @@
+import {Builder, By, until} from "selenium-webdriver";
+import {
+  getDriver,
+  getElementByXPath,
+  urls,
+  initialData,
+  executeCommand,
+} from "./utils";
+
+const firefox = require("selenium-webdriver/firefox");
+
+describe("Selenium tests for <Login />", () => {
+  let driver;
+  beforeAll(async () => {
+    jest.setTimeout(10000);
```

Comment:
again, this is completely unnecessary, please remove it, if you need to get the github actions build to wait, please use the linux `sleep` command in the github actions workflow. Try to see if 5-7 seconds are enough though. If we had to pay for these builds, this would be money thrown out of the window.